components/Cards/SchoolCard/__tests__/SchoolCard.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import SchoolCard from '../SchoolCard';
+
+const locations = [
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
+  },
+];
+
+describe('SchoolCard', () => {
+  it('should render with required props', () => {
```

Comment:
I'd like to see tests similar to `Button.test.js` line 65 "call props.onClick when button is clicked" but for the callback on both front and back card tests. The only difference is it should check for the state change too. This time, however, check for the relevant state variable to have been changed after.